### PR TITLE
refactor-me code improvements 

### PR DESCRIPTION
**Pull request by PavelPetukhov**
There are three things I changed:
1) Renamed named tuple field name from type_ to expense_name. Since 'type' is more about expenses types like variable/fixed expenses whereas here it is more just expenses items
2) Removed unnecessary parameter
3) Simplified how namedtuple is initialized

Things to consider:
1) Code won't work properly with Expense('food', 10e15 + 0.1) and min_amount=10e15 + 0.2
Since numbers are big the code will pass the ">=" check. I added epsilon (and logic for it) in one of the commits but since I didn't test all corner cases decided to removed it for this PR
2) the code could be updated to be case insensitive in case of expense names like "Foo" and "foo" are really the same

### CODE REVIEW

File: refactor_me.py [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
 from operator import itemgetter
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', 'expense_name amount')
 
 
 def sum_expenses(expenses, min_amount=0):
     aggregated_expenses = defaultdict(int)
     for expense in expenses:
-        amount = expense.amount
```
Comment (dmugtasimov) [inline refactor_me.py:12]: Things to consider for this change:
1. What is faster access a data attribute twice or put it once to a variable and then access twice from the variable?
2. If there is a exception in the code some tools like Sentry dump local variables in the stacktrace. for this case namedtuple repr would work well and we will see `expense.amount` value, but in other cases putting to a variable first would work better
3. Some IDEs like PyCharm easily highlight standalone variable usages (when you put cursor on it). Will it work for data attributes too?

File: refactor_me.py [hunk 2]
```diff
@@ -23,4 +22,4 @@ def print_expenses(expenses):
 if __name__ == '__main__':
     # TODO(dmu) HIGH: Use static fixtures and dynamic fixture framework instead
     test_expenses = (Expense('food', 4), Expense('food', 3), Expense('car', 3), Expense('dog', 1))
-    print_expenses(sum_expenses(test_expenses, 2))
+    print_expenses(sum_expenses(expenses=test_expenses, min_amount=2))
```
Comment (dmugtasimov) [inline refactor_me.py:25]: `min_amount=2` is a good point because it is keyword argument, but I would not go with `expenses=test_expenses`, because it is a positional argument.
